client/commands/v2/pysa_server.py

```diff
@@ -9,6 +9,7 @@
 version of persistent.py.
 """
 
+from pathlib import Path
```

Comment:
Don't forget to run black to sort the imports. End result should look something like this:
```
import asyncio
import logging

from pathlib import Path
from typing import List, Sequence, Dict

<REMAINING IMPORTS>
```

Reply:
I manually sorted them in a better way. I ran black before but the result was unchanged.